README.md

```diff
@@ -1,7 +1,7 @@
-# The Rust Programming Language
+<img height = "50" src = "https://img.shields.io/badge/Rust-Programming%20Language-black?style=flat&logo=rust">
```

Comment:
we should have an alt tag, I imagine

Reply:
Sure, I'll add that now

Reply:
I just previewed the changes in my mobile and it looked kinda cramped up so I set up responsiveness based on device width (height will be adjusted automatically)
I currently have it set to 90% in width of the viewing device, which you can change as per your convenience

Reply:
@steveklabnik Do the changes seem okay or should I revert/change them?